.gitignore

```diff
@@ -201,6 +201,40 @@ icu4c/source/samples/ustring/ustring.exe
 icu4c/source/stubdata/stubdatabuilt.txt
 icu4c/source/test/cintltst/cintltst
 icu4c/source/test/cintltst/cintltst.exe
+icu4c/source/test/fuzzer/break_iterator_fuzzer
```

Comment:
Does this really have anything to do with C++ static initialization? Should it really be included in this PR?